src/schema-migrate.js

```diff
@@ -7,6 +7,10 @@ module.exports = {
 
         // change the target latest schema version to migrate to here
         migrator.draft7(schemaCopy);
+        // should prepend a # before the id value to make it a valid URI
+        if (schemaCopy['$id'] && !schemaCopy['$id'].startsWith('#')) {
```

Comment:
It will set the correct the Id only to the root schema, right? As I understood, we should also correct them for the other types under `definitions`, no?